migrations/sql/R__stored_procedure_for_mine_status_information.sql

```diff
@@ -140,4 +140,4 @@ CREATE OR REPLACE FUNCTION transfer_mine_status_information() RETURNS void AS $$
             RAISE NOTICE 'Finish updating mine status in MDS';
         END;
     END;
-$$LANGUAGE plpgsql;
+$$LANGUAGE plpgsql;
```

Comment:
Had merge conflicts in this file for some reason, so I copied directly from dev, not sure why there is a dif here.